python-package/lightgbm/plotting.py

```diff
@@ -10,24 +11,36 @@
 from .sklearn import LGBMModel
 
 
-def _check_not_tuple_of_2_elements(obj, obj_name='obj'):
+def _check_not_tuple_of_2_elements(obj: Any, obj_name: str = 'obj') -> Optional[str]:
```

Comment:
```suggestion
def _check_not_tuple_of_2_elements(obj: Any, obj_name: str = 'obj') -> None:
```

This function will never return a string. It will only return `None` or throw an error.